modules/Router.js

```diff
@@ -5,7 +5,7 @@ import React from 'react'
 import createTransitionManager from './createTransitionManager'
 import { routes } from './PropTypes'
 import RouterContext from './RouterContext'
-import { createRoutes } from './RouteUtils'
+import { createRoutes, checkForUnreachableRoutes } from './RouteUtils'
```

Comment:
Since this is Dev only, I would put this in its own file to avoid impacting production builds.